cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -68,8 +68,8 @@
       "Description": "Comma delimited list of type of volume to create either new or from snapshot",
```

Comment:
Dropping it here: The cw-dashboard-substack probably needs to be updated

Reply:
Thanks for reminding! It is added